src/internal/common/apt.go

```diff
@@ -42,6 +42,13 @@ func getNonVirtualPackage(executor exec.Executor, name string) (pkg *AptPackage,
 	if isErrLine(execution.CombinedOut) {
 		return pkg, execution.Error()
 	}
+	
+	// Check if the output is just "Reverse Provides:" with no actual package info
+	trimmedOutput := strings.TrimSpace(execution.CombinedOut)
```

Comment:
There are other code locations where virtual packages are evaluated. Consolidate this logic and how it is reported.